test/test_frames.py

```diff
@@ -113,10 +113,14 @@ def test_data_frame_has_correct_flags(self):
             'END_STREAM', 'PADDED'
         ])
 
-    def test_data_frame_serializes_properly(self):
+    @pytest.mark.parametrize('data', [
+        b'testdata',
+        memoryview(b'testdata')
+        ])
```

Comment:
I don’t know how much @Lukasa cares, but one tiny nitpick is that these brackets should be dedented once to match the bracing style in the rest of the project.